tests/hazmat/primitives/test_ciphers.py

```diff
@@ -209,6 +209,15 @@ def test_invalid_backend():
         )
 
 
+def test_invalid_gcm_algorithm():
+    with raises_unsupported_algorithm(_Reasons.UNSUPPORTED_CIPHER):
+        cipher = ciphers.Cipher(
+            ARC4(b"\x00" * 16),
+            modes.GCM(b"\x00" * 12),
+        )
+        cipher.encryptor()
```

Comment:
This line is never reached, `CIpher.__init__` raises the exception AFAICT

Reply:
Yeah that's correct now that I've hoisted this into the validate alone.